usr/local/sbin/entrypoint.d/helpers/01-deskpro-opc.php.tmpl

```diff
@@ -3,8 +3,57 @@
 // OPC is behind a reverse proxy on the host machine
 // so we always trust the headers
 $CONFIG['env']['trust_proxy_data'] = [
-    '0.0.0.0/0'
+  '0.0.0.0/0'
 ];
 
-// Internal API requests should use the web task
-$CONFIG['api_urls']['base_url_private'] = {{ (getenv "DESKPRO_API_BASE_URL_PRIVATE" "http://127.0.0.1:80") | squote }};
+if (!function_exists('_opc_load_old_config_file')) {
+    function _opc_load_old_config_file(): array
+    {
+      static $opcConfig = null;
+
+      if ($opcConfig === null) {
+        $opcConfigFile = getenv('CUSTOM_MOUNT_BASEDIR').'/config/deskpro-config.php';
+        if (file_exists($opcConfigFile)) {
+          // include the other config file
+          // (the values will be scoped to this function so we don't mix it with real config)
+          include $opcConfigFile;
+          $opcConfig = $CONFIG;
+        } else {
+          $opcConfig = [];
+        }
+      }
+
+      return $opcConfig;
+    }
+}
+
+// copy/merge config set from OPC into keys on the new default config
```

Comment:
FYI @chrislrobinson - changed approach here. The `deskpro-config.php.tmpl` in this repos will always be used even on OPC. Then if it's OPC 'm just selectively overwriting certain elements.

So this'll tide us over 'til OPC can change it's config format.